puma.gemspec

```diff
@@ -1,52 +1,19 @@
 # -*- encoding: utf-8 -*-
 
-# This is only used when puma is a git dep from Bundler, so it's a little
-# weird.
+# This is only used when puma is a git dep from Bundler, keep in sync with Rakefile
 
-d = File.read(File.expand_path("../lib/puma/const.rb", __FILE__))
-if d =~ /VERSION = "(\d+\.\d+\.\d+)"/
-  version = $1
-else
-  version = "0.0.1"
-end
+version = File.read(File.expand_path("../lib/puma/const.rb", __FILE__))[/VERSION = "(\d+\.\d+\.\d+)"/, 1] || raise
```

Comment:
if this fails then something is wrong ... better fail and fix instead of having weird versions